pkg/quotaplugins/quota-forest/quota-manager/quota/quotamanager.go

```diff
@@ -39,6 +40,9 @@ type Manager struct {
 	consumerInfos map[string]*ConsumerInfo
 	// mode of the quota mnager; initially in maintenance mode
 	mode Mode
+
+	// mutex to
+	mutex sync.RWMutex
```

Comment:
@asm582, @metalcycling and @dmatch01 I believe a single mutex would be sufficient for now and MCAD use of the library. The entire quota library forest is not thread safe, we'll have to figure out how to make it so, and perhaps simplify it.

Reply:
@asm582 take a look at #392